### ISSUE TITLE
[Improvements] Player Initiative Editing

### ISSUE THREAD

**Issue by lucky-trill** [angometry/trinium-chat-buttons#9]
It would be nice to have an option to allow players to click on their initiative and be able to edit it just like GMs can. I use Initiative Double Click to get that behavior in the actual combat tracker and, in the spirit of wanting to keep options similar-ish between multiple combat trackers, I'd like to be able to allow it with the chat combat tracker too.

**Comment by angometry** [comment]
Yes! It's already in the future plans. I also want to add more options about what should and shouldn't be hidden from players.

**Comment by angometry** [comment]
Added a GM option (default off) to allow players to edit their own initiative in v1.6.0
